### ISSUE TITLE
Sublime tmThemes doesn't behave the same when converted to css & js themes in ace

### ISSUE THREAD

**Issue by narraressan** [ajaxorg/ace#3075]
I downloaded a tmTheme template from colorsublime.com and converted it to ace theme using tmtheme.js. 
Then, I used `setTheme("ace/theme/<downloaded and converted tmTheme>")` and `setMode("ace/mode/html")`. When i started the editor, the syntax color that perfectly works in sublime doesn't work in ace editor. I tried changing the mode to `setMode("ace/mode/javascript")` but still no luck. 

Any idea what I'm doing wrong? Thank you.

**Comment by nightwing** [comment]
Are there errors in browser console about missing files?
Also the automatic conversion can miss some important classes, or theme may be adding styles for scopes that are not added by ace highlighter.
If you add converted them in your fork of ace, and give me url of original tmTheme file i can check why it didn't work.

**Comment by narraressan** [comment]
I tried Flatland Monokai   ----  http://colorsublime.com/?q=flatland+monokai
I was thinking of using and editing the monokai css in the themes folder but I think it is a bad idea as users would always want to upload their own themes.

I'm really lost here...

**Comment by nightwing** [comment]
What error do you see?It seems to work well at https://github.com/nightwing/ace/commit/81880aa02cc018c45f8d697c26c87282ce755a23#diff-06e342c0f4618a790c1fcb205c373b7bR1
https://rawgit.com/nightwing/ace/81880aa02cc018c45f8d697c26c87282ce755a23/kitchen-sink.html?theme=flatland_monokai

**Comment by narraressan** [comment]
Thanks for taking the time to answer sir. Actually I was able to convert it properly using tmtheme.js but when in my sublime, html tags colors blue while in ace it colors white. I dont know what happened. Though, a while back this morning, I was able to settle things the other way. Its crude and it no longer allows users to upload their own tmtheme files but I implemented a simple in-app theme editor.

I'm really loving ace now... ^_^
